docs/source/analyze/mssql.md

```diff
@@ -0,0 +1,55 @@
+---
+title: MS SQL
+description: Check version and connection status
+---
+
+The `MS SQL` analyzer is available to check vesion and connection status of a PostgreSQL database.
```

Comment:
...of a **Microsoft SQL Server** database...

Reply:
Fixed. Appreciate the review.